### DISCUSSION TITLE
handling non-unique billing codes in the Tall CSV template

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/hospital-price-transparency#83] [Q&A] [answered]
There are "grab-bag" billing codes that are used for many different items both for supplies and for drugs.
When one parsing files using the Tall CSV template, it is necessary to identify which rows are referring to the same item, so all of the negotiated charges for the item can be gathered together.  In the Wide template all of the negotiated charges are right there on the same row. The tall format requires some reliable way to match rows up that are describing the charges for the same item.
Two examples of these types of codes are C1713 and J3490. The billing code(s) alone doesn't uniquely define an item.  The description can help, but only if the description is unique.  If there is a unique CDM code that is provided, that can also can be used.

It would be very useful for the documentation of the format or guidance to address this issue with the tall format and require either a unique description or a unique CDM (internal/local) code or both when the code(s) alone are not unique.  

Does this seem reasonable?

**Comment by RidenourK** [comment, marked as answer]
@nmayle. 

Currently, the regulation text at 180.50(b)(2) indicates that beginning July 1, 2024, each hospital must encode in its machine-readable file all standard charge information, as applicable, for each of the following required data elements including… “**a general** description of the item or service” and “**Any code(s)** used by the hospital for purposes of accounting or billing for the item or service.”

Thank you for your suggestion that we consider future regulation changes to require a unique description, code, or both when the code alone is not unique.
